source/opt/struct_packing_pass.cpp

```diff
@@ -0,0 +1,463 @@
+// Copyright (c) 2019 Google LLC
```

Comment:
Fix the copyright

Reply:
I see a mix of Google and KhronosGroup or even other companies' name in the other passes. Should I use the same as [SPIRV-Tools' license](https://github.com/KhronosGroup/SPIRV-Tools?tab=readme-ov-file#licence) or even Epic's?

Reply:
Keep the same license, but make assign the copyright to Epic in 2024.

```suggestion
// Copyright (c) 2024 Epic ...
```

I'm not sure how you do your company name officially.